be/src/formats/orc/apache-orc/c++/src/RleEncoderV2.cc

```diff
@@ -583,9 +556,7 @@ void RleEncoderV2::writePatchedBasedValues(EncodingOption& option) {
     // find the number of bytes required for base and shift it by 5 bits
     // to accommodate patch width. The additional bit is used to store the sign
     // of the base value.
-    uint32_t baseBitsNum = findClosestNumBits(option.min);
```

Comment:
wired, sr is copying orc's code directly, but this line code looks like never existed